jdi-light-bootstrap-tests/src/main/java/io/github/com/sections/navbar/NavbarContainer.java

```diff
@@ -0,0 +1,9 @@
+package io.github.com.sections.navbar;
+
+import com.epam.jdi.light.elements.composite.Section;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+
+public class NavbarContainer extends Section {
+    @UI("#navbar-containers-centred") public NavbarSimpleLinks navLinks1;
+    @UI("#navbar-containers-expanded") public NavbarComplexLinks navbarComplexLinks;
```

Comment:
Are there really the same elements as in NavbarNav.java so we could use same elements' types?